### PR TITLE
New logo added and background issues fixed

### PR DESCRIPTION
**Pull request by AdamMindGate**

### CODE REVIEW

**Review by AbdoRayan404 — COMMENTED** [review]
Can you add this logo across the website?
So in all navbars.

**Comment by AdamMindGate** [comment]
> Can you add this logo across the website? So in all navbars.

Of course.

File: views/landing.html [hunk 1]
```diff
@@ -17,50 +17,19 @@
...
-              <span>></span>
-            </a>
-
-            <a class="profile-navbar-link" onclick="logout()">
-              <img src="src/images/logout.png">
-              <p>Logout</p>
-              <span>></span>
-            </a>
-          </div>
-        </div>
```
Comment (AbdoRayan404) [inline views/landing.html:62]: What is the reason of deleting the profile navbar?
Can you bring that back?

Comment (AdamMindGate) [reply views/landing.html:62]: > What is the reason of deleting the profile navbar? Can you bring that back?

It was deleted by mistake, I will fix it.

Comment (AbdoRayan404) [reply views/landing.html]: Happens to the best of us !

**Pull request merged**